tests/ordering/tests.py

```diff
@@ -10,19 +10,25 @@
 
 
 class OrderingTests(TestCase):
-    def setUp(self):
-        self.a1 = Article.objects.create(
+
+    @classmethod
```

Comment:
The change is fine, I'm just asking that it be a be separate commit similar to 31098e3288595c13f165935f2579f1af744e5240 so that the refactoring isn't mixed in with the new feature.